package.json

```diff
@@ -47,7 +59,9 @@
     "@types/react-dom": "^17.0.0",
     "@vitejs/plugin-react": "^1.0.0",
     "autoprefixer": "^10.4.0",
+    "concurrently": "^6.4.0",
     "esbuild": "^0.13.14",
+    "nodemon": "^2.0.15",
```

Comment:
Is this being used?

Reply:
all good catches, thanks! will make sure to check before next PR 😶‍🌫️